package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@devcontainers/cli",
 	"description": "Dev Containers CLI",
-	"version": "0.57.0",
+	"version": "0.57.1",
```

Comment:
Can we revert this change? 

We prefer isolating the release changes to a separate PR (where we bump the version as well as update the changelog)

Reply:
Reverted! thanks for the advice